UI/frontend-plugins/aja-output-ui/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ target_link_libraries(
           AJA::LibAJANTV2
           $<$<PLATFORM_ID:Windows>:ws2_32.lib>
           $<$<PLATFORM_ID:Windows>:setupapi.lib>
+          $<$<PLATFORM_ID:Windows>:wbemuuid.lib>
```

Comment:
It would probably make more sense to add all these Windows DLL dependencies to the `libajantv2` target's `INTERFACE_LINK_LIBRARIES` property. That way CMake will make consumers automatically link to those and we would not need to specify them on those targets manually (as they are usage requirements of the NTV2 library).

Reply:
@paulh-aja This would require adding the Windows-specific dependency requirements into `FindLibAJANTV2.cmake` (adding them to `INTERFACE_LINK_LIBRARIES`, e.g. `$<$<PLATFORM_ID:Windows>:wbemuuid.lib>`) so that your libraries' usage requirements are contained with the generated library target.

That way any target that uses `AJA::LibAJANTV2` will automatically inherit the requirements and they don't have to be added to the targets.